tools/bake.py

```diff
@@ -1029,6 +1029,22 @@ def Scale2D( v, s, p ):
                 if not overlap_aware:
                     bpy.ops.uv.pack_islands(rotate=True, margin=margin)
                 try:  # UVP doesn't respect margins when called like this, find out why
+                    context.scene.uvpm3_props.normalize_islands = False
+                    context.scene.uvpm3_props.lock_overlapping_enable = True
+                    context.scene.uvpm3_props.lock_overlapping_mode = ('1' if
+                                                                      layer == 'CATS UV Super' else
+                                                                      '2')
+                    context.scene.uvpm3_props.pack_to_others = False
+                    context.scene.uvpm3_props.margin = margin
+                    context.scene.uvpm3_props.simi_threshold = 3
+                    context.scene.uvpm3_props.precision = 1000
+                    # Give UVP a static number of iterations to do TODO: make this configurable?
+                    for _ in range(1, 3):
+                        bpy.ops.uvpackmaster3.pack(mode_id='pack.single_tile')
+                except AttributeError:
+                    pass
```

Comment:
I'm also admittedly, not very familiar with Python. I'm not sure what the recommended practice for fall through is in this case. Please let me know if this is okay or what you advise.